spec/features/write_to_councillors_spec.rb

```diff
@@ -0,0 +1,60 @@
+require 'spec_helper'
+
+feature "Send a message to a councilor" do
+  # As someone interested in a local development application,
+  # let me write to my local councillor about it,
+  # so that I can get their help or feedback
+  # and find out where they stand on this development I care about
+
+  context "when not logged in" do
+    background do
+      authority = create(:authority, full_name: "Foo")
+      VCR.use_cassette('planningalerts') do
+        application = create(:application, id: "1", authority_id: authority.id, comment_url: 'mailto:foo@bar.com')
+        visit application_path(application)
+      end
+    end
+
+    scenario "can’t see councilor messages sections" do
+      page.should_not have_content("Write to your elected councillors about this application")
+      # and you should not be able to write and submit a message.
+    end
+  end
+
+  context "when logged in as admin" do
+    background do
+      authority = create(:authority, full_name: "Foo")
+      admin = create(:admin)
+
+      visit new_user_session_path
+      within("#new_user") do
+        fill_in "Email", with: admin.email
+        fill_in "Password", with: admin.password
+      end
+      click_button "Sign in"
+      expect(page).to have_content "Signed in successfully"
+
+      VCR.use_cassette('planningalerts') do
+        application = create(:application, id: "1", authority_id: authority.id, comment_url: 'mailto:foo@bar.com')
+        visit application_path(application)
```

Comment:
This action should be in the scenario because it's part of that process I think.